app/controllers/ems_cloud_controller.rb

```diff
@@ -19,6 +19,11 @@ def self.table_name
     @table_name ||= "ems_cloud"
   end
 
+  def show
+    @breadcrumbs = [{:name => "Cloud Providers", :url => "/ems_cloud/show_list"}]
```

Comment:
Should not be it _('Cloud Providers')? (Same below) :globe_with_meridians:

Reply:
Ye, it should